AnaProd/anaTupleProducer.py

```diff
@@ -69,13 +70,23 @@ def addAllVariables(dfw, syst_name, isData, trigger_class):
     for var in ["covXX", "covXY", "covYY"]:
         dfw.DefineAndAppend(f"met_{var}", f"static_cast<float>(MET_{var})")
 
+    dfw.Define(f"Additional_jet_p4", "Jet_p4[AdditionalJet_B1]")
+    dfw.DefineAndAppend(f"AdditionalJet_pt", f"Jet_pt[AdditionalJet_B1]")
```

Comment:
- those are original nanoAOD values. You need to store the shifted ones.
- add static_cast<float> where needed

Reply:
ok